tsdef/types.d.ts

```diff
@@ -98,8 +98,6 @@ export type BandwidthProfileMode = 'grid' | 'collaboration' | 'presentation';
 export type VideoContentPreferencesMode = 'auto' | 'manual';
 export type ClientTrackSwitchOffControl = 'auto' | 'manual';
 
-export type TwilioWarning = 'recording-media-lost';
```

Comment:
I think its good to have the specific type for this. Maybe we can rename it to `NotifyWarning`